.env.production

```
@@ -39,7 +39,7 @@ REACT_APP_DOMAIN_REGEX_LOCAL="^(:?localhost:\d{2,5}|(?:127|192)(?:\.[0-9]{1,3}){
 REACT_APP_DOMAIN_REGEX_PR="^pr\d+--cowswap\.review"
 REACT_APP_DOMAIN_REGEX_DEV="^cowswap\.dev"
 REACT_APP_DOMAIN_REGEX_STAGING="^cowswap\.staging"
-REACT_APP_DOMAIN_REGEX_PROD="^cowswap\.exchange$"
+REACT_APP_DOMAIN_REGEX_PROD="(^cowswap\.exchange$) | (^swap\.cow\.fi$)"
 REACT_APP_DOMAIN_REGEX_BARN="^barn\.cowswap\.exchange$"
```

Not there yet, but since we are applying this change, we should also add the barn alternative here

```suggestion
REACT_APP_DOMAIN_REGEX_BARN="^barn\.(cowswap\.exchange|swap\.cow\.fi)$"
```

Also potentially barn.cow.fi? Unsure

---

Don't you need additional groups for this:
```
REACT_APP_DOMAIN_REGEX_BARN="^barn\.(cowswap\.exchange)|(swap\.cow\.fi)$"
```

---

Yep, we are not capturing the strings, just matching them

---

I got confused about "precedence" of `|`. RegExp is basically black magic to me.